app/controllers/items_controller.rb

```diff
@@ -29,14 +29,17 @@ def index                                   #items in library
 
   def admin_index                             #denied and pending items in library
     @items = search
-
-    @pendings, @denieds = [], []
+    
+    @pendings, @denieds, @approveds = [], [], []
 
     @items.each do |item|
       @pendings << item if item.status == Item::PENDING
       @denieds << item if item.status == Item::DENIED
+      @approveds << item if item.status == Item::APPROVED
     end
 
+    @reports = Report.where(status: Report::PENDING)
```

Comment:
This feels like it should be in a different branch

Reply:
yes I merged with quang/report-form so the UI is completely updated for the demo

Reply:
His branch is a PR request too, but needs some changes. For the purposes of the demo, this will have to do. But his will need to be changed and approved.